extensions/ql-vscode/package.json

```diff
@@ -2068,7 +2081,8 @@
     "ts-loader": "^9.4.2",
     "ts-node": "^10.7.0",
     "ts-unused-exports": "^10.0.0",
-    "typescript": "^5.0.2"
+    "typescript": "^5.0.2",
+    "unzipper": "^0.10.14"
```

Comment:
I haven't looked at the full PR yet, but is there a reason we need to be using `unzipper` here? We do already have [`yauzl`](https://github.com/thejoshwolfe/yauzl) as a dependency and some helpers functions in [`unzip.ts`](https://github.com/github/vscode-codeql/blob/96b7722f260b44c75b8f8da309fe0b7e8fa30b85/extensions/ql-vscode/src/common/unzip.ts) and [`unzip-concurrently.ts`](https://github.com/github/vscode-codeql/blob/main/extensions/ql-vscode/src/common/unzip-concurrently.ts).

Reply:
Thanks. I wasn't aware of those dependencies. I can switch.